misc/docerina/src/main/java/org/ballerinalang/docgen/Generator.java

```diff
@@ -39,6 +39,7 @@
 import org.ballerinalang.model.tree.NodeKind;
```

Comment:
Is there a possibility to add a unit test for this?

Reply:
Added a test case